api-spec.yaml

```diff
@@ -0,0 +1,134 @@
+consumes:
+- application/json
+info:
+  description: Column - Ansible as a service
+  title: Column
+  version: 0.0.1
+  contact:
+    website: "https://github.com/vmware/column"
+  license:
+    name: "BSD-2"
+    url: "https://github.com/vmware/column/blob/master/LICENSE"
+produces:
+- application/json
+swagger: "2.0"
+schemes:
+- http
+basePath: /v1
+paths:
+  /tasks:
```

Comment:
I think "tasks" might be confused with Ansible tasks in a playbook.  Rather some other noun not already used such as "job", "run", etc